scripts/model.js

```diff
@@ -45,11 +45,11 @@ class Data { // eslint-disable-line
     /** @type {Date} updated */
     this.updated = new Date(jsonData.updated)
     /** @type {Project[]} _projects */
-    let _projects = []
-    for(let each of jsonData.projects) {
-      let project = new Project(each.type, each.name, each.link, each.media, each.description, each.date)
-      _projects.push(project)
-    }
+    let _projects = jsonData.projects.map(
+      /** @param {Object} each */
+      each => {
+        return new Project(each.type, each.name, each.link, each.media, each.description, each.date)
```

Comment:
I probably do not need a return here but I'd like it because it is more descriptive.